raw_data/preprocessed/africa/guinea-bissau_portuguese.txt

```diff
@@ -0,0 +1,199 @@
 desenvolvimento durável dos mesmos; Lixos Perigosos são substâncias que contêm características de riscos por serem inflamáveis, explosivas, corrosivas, tóxicas, infecciosas ou radioactivas, ou por apresentarem qualquer outra característica que constitua perigo para a vida ou saúde das pessoas para a qualidade do ambiente; Nicho Ecológico é a posição ou a situação duma espécie no interior do seu ecossistema, incluindo a sua nutrição, os tempos de alimentação, temperatura, humidade e modo como se reproduz; Ofensa Ecológica é todo o acto ou facto humano, culposo ou não que tenha como resultado a produção de um dano nos componentes ambientais protegidas pela presente lei; Ordenamento do Território é o processo integrado de organização do espaço biofísico, tendo como objectivo ocupação, uso e transformação do território de acordo com as suas capacidades e vocações, e a permanência dos valores de equilíbrio biológico e estabilidade ecológica, numa perspectiva de aumento da sua capacidade suporte da vida; Poluição é a deposição no ambiente de substâncias gasosas, líquidas ou sólidas ou de várias formas de energia provocadas pelas actividades humanas; Qualidade de Vida é o resultado da interacção de múltiplos factores no funcionamento das sociedades humanas que se traduz no bem-estar físico, mental e social e na afirmação cultural do indivíduo; Qualidade do Ambiente é o equilíbrio e a sanidade do ambiente, incluindo a adequação dos seus componentes às necessidades do homem e de outros seres vivos; Queimada é a prática agrícola rudimentar, que consiste na destruição da vegetação natural, quase sempre matas, com o fim de preparar o terreno para semear ou plantar; essa prática prejudica a fertilidade do solo pela liberação de sais minerais; Resíduos são substâncias ou objectos que se eliminam ou que se tem a intenção de eliminar, ou que se é obrigado por lei a eliminar; Seca é o fenómeno que ocorre naturalmente quando a precipitação registada é significativamente inferior aos valores normais, provocando um sério desequilíbrio hídrico que afecta negativamente os sistemas de produção dependentes dos recursos da terra; Saneamento Básico é a solução dos problemas relacionados estritamente com abastecimento de água e disposição dos esgotos de uma comunidade, como da inclusão do lixo e outros problemas que terminarão por tomar as acções, serviços e obras considerados prioritários em programas de saúde pública, nomeadamente o abastecimento público de água, remoção e o tratamento de esgoto; Terra é o sistema bio-produtivo terrestre que compreende o solo, a vegetação, outros componentes do biota e os processos ecológicos e hidrológicos que se desenvolvem dentro do sistema; Zonas Húmidas são áreas de pântano ou água, natural ou artificial, permanente ou temporária, parada ou corrente, doce, salobra ou salgada, incluindo as águas do mar cuja profundidade na maré baixa não excede seis metros, que sustentam animal e que requeira condições de saturação aquática do solo.
```

Comment:
As per the legislation, "; essa prática" should be ". essa prática". However, I see that you've intentionally changed this. Just wanted to know why this is the case for amendments to future legislations.

Reply:
Similarly to the legislation from Benin, it seemed inappropriate to interrupt the list partway through with a period.

Reply:
Noted, thank you for the explanation.